app/models/expense.rb

```diff
@@ -1,3 +1,28 @@
 class Expense < ApplicationRecord
   belongs_to :tour
+
+
+  require 'csv'
+
+  def self.import(file)
```

Comment:
I just want to make sure you understand how this works - happy to talk about it

Reply:
thanks for the walk through